offix/src/main/java/org/aerogear/offix/Offix.kt

```diff
@@ -11,7 +11,21 @@ import com.apollographql.apollo.api.Mutation
 import com.apollographql.apollo.api.Operation
 import com.apollographql.apollo.api.Response
 import com.apollographql.apollo.exception.ApolloException
+import com.google.gson.Gson
+import okhttp3.Interceptor
 import org.json.JSONObject
+import java.nio.charset.Charset
+
+/*
+Global variable which will keep track of whether the conflict is detected or not.
+In case any conflicts come, the error string would be stored in this variable.
+ */
+lateinit var conflcitDetected: String
```

Comment:
String?

Reply:
Internally it's json but from inside the interceptor we can get response as string only.